cli/cmd/images/image.go

```diff
@@ -0,0 +1,123 @@
+package images
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/urfave/cli"
+
+	"github.com/rancher/wrangler/pkg/kv"
+
+	"github.com/rancher/rio/cli/pkg/clicontext"
+	"github.com/rancher/rio/cli/pkg/localbuilder"
+	"github.com/rancher/rio/cli/pkg/tables"
+	"github.com/rancher/rio/pkg/constants"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type catalog struct {
+	Repositories []string `json:"repositories,omitempty"`
+}
+
+type repository struct {
+	Name string   `json:"name,omitempty"`
+	Tags []string `json:"tags,omitempty"`
+}
+
+type Image struct {
+	Namespace string
+	Repo      string
+	Tag       string
+	Image     string
+}
+
+type Images struct {
+}
+
+func (i *Images) Customize(cmd *cli.Command) {
+	cmd.ShortName = "image"
+}
+
+func (i *Images) Run(ctx *clicontext.CLIContext) error {
+	pods, err := ctx.Core.Pods(ctx.SystemNamespace).List(metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	for _, pod := range pods.Items {
+		if strings.HasPrefix(pod.Name, constants.BuildkitdService) {
+			readyChan := make(chan struct{})
+			go func() {
+				if err := localbuilder.PortForward(ctx.K8s, "9998", "80", pod, false, readyChan, localbuilder.ChanWrapper(ctx.Ctx.Done())); err != nil {
```

Comment:
Hardcoding this and URLs below feels kinda gross. Could use constants but that doesn't help all that much. Is there a reason we use both `localhost` and `127.0.0.1` ?

Reply:
I will change that to constants. I think for image we always use localhost since it is better for image name. But for url we do url to avoid dns look up.